frontend/src/components/BetaAlert.tsx

```diff
@@ -1,39 +1,51 @@
 "use client";
 
-import { Trans, useTranslation } from "next-i18next";
+import Link from "next/link";
 
 import { ExternalRoutes } from "src/constants/routes";
 import FullWidthAlert from "./FullWidthAlert";
 
-const BetaAlert = () => {
-  const { t } = useTranslation("common", {
-    keyPrefix: "Beta_alert",
-  });
+// TODO: Remove for i18n update.
+type BetaStrings = {
+  alert_title: string;
+  alert: string;
+};
+
+type Props = {
+  beta_strings: BetaStrings;
+};
+
+const BetaAlert = ({ beta_strings }: Props) => {
+  // TODO: Remove for i18n update.
+  const title_start = beta_strings.alert_title.substring(
+    0,
+    beta_strings.alert_title.indexOf("<LinkToGrants>"),
+  );
+  const title_end = beta_strings.alert_title.substring(
+    beta_strings.alert_title.indexOf("</LinkToGrants>") +
+      "</LinkToGrants>".length,
+  );
+  const link = (
+    <>
+      {title_start}
+      <Link
+        target="_blank"
+        rel="noopener noreferrer"
+        href={ExternalRoutes.GRANTS_HOME}
+      >
+        www.grants.gov
+      </Link>
+      {title_end}
+    </>
+  );
```

Comment:
This has a smell. But it gets us there. Would get real messy with multiple links in a string.

Reply:
Agreed. This should be fixed in #1657